clang/lib/Sema/SemaExpr.cpp

```diff
@@ -257,15 +257,16 @@ bool Sema::DiagnoseUseOfDecl(NamedDecl *D, ArrayRef<SourceLocation> Locs,
                              diag::err_esimd_global_in_sycl_context,
                              Sema::DeviceDiagnosticReason::Sycl);
     } else if (auto *FDecl = dyn_cast<FunctionDecl>(D)) {
-      // SYCL device function cannot be called from an ESIMD context. However,
-      // funcitons that start with '__spirv_' or '__sycl_' are exceptions to
-      // this rule.
+      // SYCL device function cannot be called from ESIMD context. However,
+      // there are some exceptions from this rule - funcitons that start
```

Comment:
Nit: functions*

Reply:
Thank you!
I fixed this misprint here: https://github.com/intel/llvm/pull/9615/commits/06f1c4c31f3fb16126e9e79ebb454cd701af2555